components/Button/Button.tsx

```diff
@@ -0,0 +1,19 @@
+import { MouseEventHandler } from 'react';
+import { clsx as cx } from 'clsx';
+
+type ButtonProps = {
+  name: string;
+  className?: string;
+  onClick: MouseEventHandler<HTMLButtonElement>;
+};
+
+export const Button = ({ name, className, onClick }: ButtonProps) => {
+  return (
+    <button
+      className={cx('align-middle mx-auto text-sm leading-5 font-semibold text-white', className)}
```

Comment:
Why do you use `vertical-align: middle` instead of `vertical-align: baseline`? Button does not include paddings or colors as it should, in general it could have some variants: `sm, md, lg` and color variants `blue, indigo`, which I prefer more than raw css classes. A button shouldn't have margin properties - if you like to add it, I think it should be added via `className` prop.

```suggestion
      className={cx('align-middle text-sm leading-5 font-semibold text-white', className)}
```